### PR TITLE
Chore: Add feedback button to the browser extension

### LINKED ISSUE #2525: [extension-browser] Add feedback button in browser extension [0.5]

### PR DESCRIPTION
**Pull request by sarvaje**
Fix #2525

## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [ ] Added/Updated related tests.

### CODE REVIEW

**Comment by sarvaje** [comment]
This is how it looks:
image
image

File: packages/extension-browser/src/_locales/en/messages.json [hunk 1]
```diff
@@ -186,5 +186,9 @@
     "targetUrl": {
         "message": "URL: $1",
         "description": "Label for the result page with the url analyzed."
+    },
+    "feedback": {
+        "message": "Feedback",
+        "description": "Label for the feedback button"
```
Comment (antross) [inline packages/extension-browser/src/_locales/en/messages.json]: Nit: I know the list isn't perfectly alphabetized, but if we can keep this roughly in alphabetical order I think that will help us find strings more easily in the future.

File: packages/extension-browser/src/devtools/views/controls/feedback.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+const openIssueUrl = 'https://github.com/webhintio/hint/issues/new?labels=type%3Abug&amp;template=2-bug-report-browser.md&amp;title=%5BBug%5D+Bug+description';
+
+/**
+ * Link to github to give feedback.
+ */
+const Feedback = () => {
+    return (
+        <span className={styles.root}>
+            <ExternalLink href={openIssueUrl}>
```
Comment (antross) [inline packages/extension-browser/src/devtools/views/controls/feedback.tsx]: Nit: You can drop the wrapping span and put the class directly on `ExternalLink` if you want.
```suggestion
            <ExternalLink className={styles.root} href={openIssueUrl}>
```

That said, I've been trying to follow the best practice where parent components control layout, not children. In that case I'd actually prefer if this component accepted a className and the position were defined as part of the parent component's stylesheet (even if that proves to be slightly redundant at the moment).

File: packages/extension-browser/src/devtools/views/pages/config/header.tsx [hunk 3]
```diff
@@ -5,6 +5,7 @@ import { getMessage } from '../../../utils/i18n';
 import Button from '../../controls/button';
 
 import * as styles from './header.css';
+import Feedback from '../../controls/feedback';
```
Comment (antross) [inline packages/extension-browser/src/devtools/views/pages/config/header.tsx]: Move the import below `import Button`.

File: packages/extension-browser/src/devtools/views/pages/config/header.tsx [hunk 4]
```diff
@@ -15,6 +16,7 @@ const ConfigHeader = () => {
             <Button type="submit" primary={true}>
                 {getMessage('startScanButtonLabel')}
             </Button>
+            <Feedback></Feedback>
```
Comment (antross) [inline packages/extension-browser/src/devtools/views/pages/config/header.tsx]:
```suggestion
            <Feedback />
```

File: packages/extension-browser/src/devtools/views/controls/feedback.tsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+import ExternalLink from './external-link';
+
+import * as styles from './feedback.css';
+
+const openIssueUrl = 'https://github.com/webhintio/hint/issues/new?labels=type%3Abug&amp;template=2-bug-report-browser.md&amp;title=%5BBug%5D+Bug+description';
+
+/**
+ * Link to github to give feedback.
+ */
+const Feedback = () => {
```
Comment (antross) [inline packages/extension-browser/src/devtools/views/controls/feedback.tsx]: Nit: I think `FeedbackLink` (with a filename of `feedback-link.tsx`) would be more consistent.

File: packages/extension-browser/src/devtools/views/pages/results/header.tsx [hunk 6]
```diff
@@ -7,6 +7,7 @@ import Button from '../../controls/button';
 import Label from '../../controls/label';
 import LabelText from '../../controls/label-text';
 import Toggle from '../../controls/toggle';
+import Feedback from '../../controls/feedback';
```
Comment (antross) [inline packages/extension-browser/src/devtools/views/pages/results/header.tsx]: Nit: This should go after `import Button` to keep the list alphabetical.

File: packages/extension-browser/src/devtools/views/pages/results/header.tsx [hunk 7]
```diff
@@ -46,6 +47,7 @@ const ResultsHeader = ({ onConfigureClick, showPassed, setShowPassed, url }: Pro
                         {getMessage('showPassedHintsLabel')}
                     </LabelText>
                 </Label>
+                <Feedback></Feedback>
```
Comment (antross) [inline packages/extension-browser/src/devtools/views/pages/results/header.tsx]:
```suggestion
                <Feedback />
```

**Comment by sarvaje** [comment]
@antross done!

File: packages/extension-browser/src/devtools/views/controls/feedback-link.tsx [hunk 8]
```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+import { getMessage } from '../../utils/i18n';
+
+import ExternalLink from './external-link';
+
+import * as styles from './feedback.css';
```
Comment (molant) [inline packages/extension-browser/src/devtools/views/controls/feedback-link.tsx]:
```suggestion
import * as styles from './feedback-link.css';
```

**Comment by sarvaje** [comment]
@antross this is ready for a review!

**Pull request merged**